xbmc/video/VideoInfoScanner.cpp

```diff
@@ -1688,6 +1668,28 @@ namespace KODI::VIDEO
     return type;
   }
 
+  std::pair<CInfoScanner::INFO_TYPE, std::unique_ptr<IVideoInfoTagLoader>> CVideoInfoScanner::
+      ReadInfoTag(CFileItem& item, const ADDON::ScraperPtr& scraper, bool lookInFolder)
+  {
+    auto result = CInfoScanner::NO_NFO;
+    std::unique_ptr<IVideoInfoTagLoader> loader(
+        CVideoInfoTagLoaderFactory::CreateLoader(item, scraper, lookInFolder));
+    if (loader)
+    {
+      CVideoInfoTag& infoTag = *item.GetVideoInfoTag();
+      infoTag.Reset();
+      result = loader->Load(infoTag, false);
+
+      // keep some properties only if advancedsettings.xml says so
+      const auto advancedSettings = CServiceBroker::GetSettingsComponent()->GetAdvancedSettings();
+      if (!advancedSettings->m_bVideoLibraryImportWatchedState)
+        infoTag.ResetPlayCount();
+      if (!advancedSettings->m_bVideoLibraryImportResumePoint)
+        infoTag.SetResumePoint(CBookmark());
```

Comment:
may not make much difference, the advanced settings could be retrieved once in CVideoInfoScanner()?

Reply:
OK, I did that in a separate commit which I won't include in the backport.